src/plugins/data_source_management/framework/utils/shared.ts

```diff
@@ -0,0 +1,346 @@
+export const observabilityIntegrationsTitle = 'Integrations';
+export const observabilityIntegrationsPluginOrder = 9020;
+
+export const observabilityDataConnectionsID = 'datasources';
+export const observabilityDataConnectionsTitle = 'Data sources';
+export const observabilityDataConnectionsPluginOrder = 9030;
+
+export const queryWorkbenchPluginID = 'opensearch-query-workbench';
+export const queryWorkbenchPluginCheck = 'plugin:queryWorkbenchDashboards';
+
+// Shared Constants
+export const SQL_DOCUMENTATION_URL = 'https://opensearch.org/docs/latest/search-plugins/sql/index/';
+export const PPL_DOCUMENTATION_URL =
+  'https://opensearch.org/docs/latest/search-plugins/sql/ppl/index';
+export const PPL_PATTERNS_DOCUMENTATION_URL =
+  'https://github.com/opensearch-project/sql/blob/2.x/docs/user/ppl/cmd/patterns.rst#description';
+export const UI_DATE_FORMAT = 'MM/DD/YYYY hh:mm A';
+export const PPL_DATE_FORMAT = 'YYYY-MM-DD HH:mm:ss.SSSSSS';
+export const OTEL_DATE_FORMAT = 'YYYY-MM-DDTHH:mm:ss';
+export const SPAN_REGEX = /span/;
+
+export const PROMQL_METRIC_SUBTYPE = 'promqlmetric';
+export const OTEL_METRIC_SUBTYPE = 'openTelemetryMetric';
+export const PPL_METRIC_SUBTYPE = 'metric';
+
+export const PPL_SPAN_REGEX = /by\s*span/i;
+export const PPL_STATS_REGEX = /\|\s*stats/i;
+export const PPL_INDEX_INSERT_POINT_REGEX = /(search source|source|index)\s*=\s*([^|\s]+)(.*)/i;
+export const PPL_INDEX_REGEX = /(search source|source|index)\s*=\s*([^|\s]+)/i;
+export const PPL_WHERE_CLAUSE_REGEX = /\s*where\s+/i;
+export const PPL_NEWLINE_REGEX = /[\n\r]+/g;
+export const PPL_DESCRIBE_INDEX_REGEX = /(describe)\s+([^|\s]+)/i;
+
+// Observability plugin URI
+const BASE_OBSERVABILITY_URI = '/_plugins/_observability';
+const BASE_DATACONNECTIONS_URI = '/_plugins/_query/_datasources';
+export const OPENSEARCH_PANELS_API = {
+  OBJECT: `${BASE_OBSERVABILITY_URI}/object`,
+};
+export const OPENSEARCH_DATACONNECTIONS_API = {
+  DATACONNECTION: `${BASE_DATACONNECTIONS_URI}`,
+};
+
+// Saved Objects
+export const SAVED_OBJECT = '/object';
+
+// Color Constants
+export const PLOTLY_COLOR = [
+  '#3CA1C7',
+  '#54B399',
+  '#DB748A',
+  '#F2BE4B',
+  '#68CCC2',
+  '#2A7866',
+  '#843769',
+  '#374FB8',
+  '#BD6F26',
+  '#4C636F',
+];
+
+export const LONG_CHART_COLOR = PLOTLY_COLOR[1];
+
+export const pageStyles: CSS.Properties = {
+  float: 'left',
+  width: '100%',
+  maxWidth: '1130px',
+};
+
+export enum VIS_CHART_TYPES {
+  Bar = 'bar',
+  HorizontalBar = 'horizontal_bar',
+  Line = 'line',
+  Pie = 'pie',
+  HeatMap = 'heatmap',
+  Text = 'text',
+  Histogram = 'histogram',
+}
+
+export const NUMERICAL_FIELDS = ['short', 'integer', 'long', 'float', 'double'];
+
+export const ENABLED_VIS_TYPES = [
+  VIS_CHART_TYPES.Bar,
+  VIS_CHART_TYPES.HorizontalBar,
+  VIS_CHART_TYPES.Line,
+  VIS_CHART_TYPES.Pie,
+  VIS_CHART_TYPES.HeatMap,
+  VIS_CHART_TYPES.Text,
+];
+
+// Live tail constants
+export const LIVE_OPTIONS = [
```

Comment:
same here, you would not need these as well, these are live tails stuff used in observability

Reply:
removed